packages/plugin-commands-installation/test/import.ts

```diff
@@ -72,14 +74,37 @@ test('import from yarn.lock', async () => {
 
   const project = assertProject(process.cwd())
   const lockfile = await project.readLockfile()
-  expect(lockfile.packages).toHaveProperty(['/dep-of-pkg-with-1-dep/100.1.0'])
-  expect(lockfile.packages).not.toHaveProperty(['/dep-of-pkg-with-1-dep/100.0.0'])
+  expect(lockfile.packages).toHaveProperty(['/dep-of-pkg-with-1-dep/100.0.0'])
+  expect(lockfile.packages).toHaveProperty(['/dep-of-pkg-with-1-dep/101.0.0'])
+  expect(lockfile.packages).not.toHaveProperty(['/dep-of-pkg-with-1-dep/100.1.0'])
 
   // node_modules is not created
   await project.hasNot('dep-of-pkg-with-1-dep')
   await project.hasNot('pkg-with-1-dep')
 })
 
+test.only('import from nested yarn.lock with an existing root pnpm-lock', async () => {
```

Comment:
This is a new test case that demonstrates the fix for problem 1 discussed in [this comment](https://github.com/pnpm/pnpm/issues/4385#issuecomment-1072882829).

Reply:
I removed the `.only` which I accidentally left in